downstream/assemblies/platform/assembly-edge-manager-intro.adoc

```diff
@@ -0,0 +1,19 @@
+[id="assembly-edge-manager-intro"]
+
+= {RedHatEdge} overview
+
+{RedHatEdge} provides streamlined management of edge devices and applications through a declarative approach. 
+By defining the required state of your edge devices, which includes your operating system versions, host configurations, and application deployments, Edge Manager automatically implements and maintains these configurations across your entire device fleet.
+
+// Confirm what is actually being implemented
+
+// include::platform/con-edge-manager-core-capabilities.adoc[leveloffset=+1]
+
+// == Integration capabilities
+
+// The Edge Manager aims to offer the following:
+
+// * Support for external authentication providers such as Keycloak, OIDC, and OpenShift Authentication API
```

Comment:
For the ACM integration, we are implementing Kubernetes / OpenShift Authentication. We have implemented auth via Keycloak/OIDC, too, but it's not well tested.